versioned_docs/version-v1.4/upgrade/v1-1-2-to-v1-2-0.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 11
```

Comment:
I assumed:  previously, it was intentional to put the `not recommended` after the `troubleshooting`; 

but it looks to be a mistake, both `v-112-to-v120` and `troubleshooting` has `postion 9` before PR 743

Reply:
Yeah, I don't remember there's such a convention. Thank you for clarifying.